src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -422,19 +519,32 @@ public enum RisingRivals implements LogicCardInfo {
         return basic (this, hp:HP090, type:FIGHTING, retreatCost:1) {
           weakness G
           move "Trample", {
-            text ". If that coin flip is heads, this attack does 30 damage to that Pokémon."
+            text "Flip a coin for each Benched Pokémon. If that coin flip is heads, this attack does 30 damage to that Pokémon."
             energyCost F, C
             attackRequirement {}
             onAttack {
-              damage 0
+              my.bench.each {
+                bc"$it"
```

Comment:
The damage should already state the Pokémon. You could instead make this part of the info string for the flip though. Broadcasting just the Pokémon name here is useless log spam though.